### PR TITLE
Expose setup stanza

### PR DESCRIPTION
**Pull request by rnjtranjan**

### CODE REVIEW

**Comment by rnjtranjan** [comment]
Fix:https://github.com/composewell/streamly/issues/2051

File: core/src/DocTestDataFileSystem.hs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{- $setup
+>>> :m
+>>> import qualified Streamly.Data.Array as Array
+>>> import qualified Streamly.Data.Fold as Fold
+>>> import qualified Streamly.Data.Unfold as Unfold
+>>> import qualified Streamly.Data.Stream as Stream
```
Comment (harendra-kumar) [inline core/src/DocTestDataFileSystem.hs:6]: Sort the modules.

File: core/src/DocTestDataUnicode.hs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+>>> :set -XOverloadedStrings
+
+>>> import Streamly.Internal.Unicode.Stream
+>>> import Streamly.Internal.Unicode.String
+
+>>> import qualified Streamly.Data.Fold as Fold
+>>> import qualified Streamly.Data.Stream as Stream
+>>> import qualified Streamly.Internal.Unicode.Array as Array
+>>> import qualified Streamly.Internal.Unicode.Stream as Unicode
+>>> import qualified Streamly.Unicode.Parser as Unicode
```
Comment (harendra-kumar) [inline core/src/DocTestDataUnicode.hs:14]: Separate non-internal imports from internal ones. Add a comment before internal imports - see Streamly.Data.Stream.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]
Need to include the same setup stanza in corresponding released modules as well. Otherwise there is not much use of this.

File: core/src/DocTestDataUnicode.hs (hunk 2 above)
Comment (harendra-kumar) [reply core/src/DocTestDataUnicode.hs:12]: Do this in other modules as well.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]
Need to include the same setup stanza in corresponding released modules as well. Otherwise there is not much use of this.

**Review by harendra-kumar — COMMENTED** [review]
1. We need this only for exposed modules.
2. We should not combine setup of multiple exposed modules together.

**Pull request merged**